lib/models/ynab/payee.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Models
+  module YNAB
+    class Payee
+      include ActiveModel::Model
+
+      attr_accessor :name, :id
+
+      validate :payee_info?
```

Comment:
Maybe u could simplify

```suggestion
      validates :name, presence: true, if: id
      validates :id, presence: true, if: :name
```

Reply:
does this do the same?
I'm confused